src/test/java/com/cook4me/controller/OfferManagerTest.java

```diff
@@ -0,0 +1,78 @@
+package com.cook4me.controller;
+
+import com.cook4me.model.Offer;
+import org.junit.*;
+import static org.junit.Assert.*;
+
+
+/**
+ * Created by Gaba on 2016-12-08.
+ */
+
+public class OfferManagerTest {
+    private Offer offer;
+    private OfferManager offerManager;
+    private Offer nullOffer;
+    private Offer almostNullOffer;
+
+    @Before
+    public void setUp(){
+        offerManager = new OfferManager();
+        offer = new Offer("bigos", "Krakow", "20.12.2016", "yummy stuff", "880880880");
+        nullOffer = null;
+        almostNullOffer = new Offer(null, null, null, null, null);
+
+    }
+
+    @Test
+    public void testStoreOffer(){
```

Comment:
name of test method should describe requirement which is checked in test. e.g.: storedOfferFiledsShouldNotBeChenged or offerManegerShouldGenerateAndSetOfferId
word "test" is not needed